util/command_parser.go

```diff
@@ -0,0 +1,25 @@
+package util
+
+import (
+	"flag"
+	"fmt"
+)
+
+func ParseCommand(command []string) (map[string]string, error) {
+	fs := flag.NewFlagSet(command[0], flag.ExitOnError)
+	var (
+		ymlPath = fs.String("f", "default-value", "Path of yaml to publish")
```

Comment:
"default-value" should be "./command_spec.yml"? or I think other file name is better.
https://screwdriver-cd.slack.com/archives/C8F6DMR7W/p1513318926000022

Reply:
I've adopt ```sd-command.yaml``` like catto said"

Reply:
It's a design change so you should also send PR to [design/commands.md](https://github.com/screwdriver-cd/screwdriver/blob/master/design/commands.md)

Reply:
PR to desigin is [here](https://github.com/screwdriver-cd/screwdriver/pull/895).